### ISSUE TITLE
Arg to auto close REPL after running script

### ISSUE THREAD

**Issue by jameshulse** [scriptcs/scriptcs#482]
When running a script via scriptcs {file} -repl it would be nice to be able to pass another argument such as -autoclose which automatically exits scriptcs once it has completed running the file. It would also be handy to be able to mute the extra output such as "INFO: Loading preseeded script:" etc.

This would be useful for the sublime text plugin to allow for running simple scripts without having to use Console.WriteLine for output - we would benefit from the repl auto-dumping.

Basically I want to use sublime text to fiddle with simple on line statements - the command line isn't great for editing.
